xml/admin_ceph_tips.xml

```diff
@@ -859,13 +859,28 @@ disk_led:
   <para>
    The default configuration for <filename>disk_led.sls</filename> offers disk
    LED support through the <systemitem>libstoragemgmt</systemitem> layer.
-   However, <systemitem>libstoragemgmt</systemitem> provides this support
-   through a hardware-specific plug-in and third party tools. Unless both the
-   <systemitem>libstoragemgmt</systemitem> plug-in and the third party tools
-   appropriate for the hardware are installed,
-   <systemitem>libstoragemgmt</systemitem> will not be able to adjust the LEDs.
+   <systemitem>libstoragemgmt</systemitem> provides this support
```

Comment:
Capital L?

Reply:
For libstoragemgmt?

Reply:
Dropping the _However_ made it the beginning of the sentence.   I know, it's a library name.

Reply:
Yeah, we don't need to start the sentence with a capital letter if it's a name of something (as such). I dropped the _however_ as I didn't think it was necessary.

Reply:
I would try to avoid starting a sentence with a lower-case letter, even if it is permitted when it's part of a filename or something. (Same as back in the day it was editorial advice to try to avoid forming a plural of "Pentium" because both "Pentiums" and "Pentia" look weird.)

E.g.

The default configuration for <filename>disk_led.sls</filename> offers disk
 LED support through <systemitem>libstoragemgmt</systemitem>.
The <systemitem>libstoragemgmt</systemitem> layer provides this support...